config/routes.rb

```diff
@@ -82,6 +82,9 @@
   mount Peek::Railtie => '/peek'
   mount Riiif::Engine => '/images', as: 'riiif'
 
-
+  require 'sidekiq/web'
+  authenticate :user, lambda {|u| u.roles_name.include? 'admin' } do
+    mount Sidekiq::Web => '/sidekiq'
```

Comment:
@mankind where would this URL be available in the Hyku application?

Reply:
At the tenant level or at the root level. It is available on both as far as you are logged in as admin.
http://library.localhost:8080/sidekiq  or http://localhost:8080/sidekiq

Reply:
@fradeve sorry, a slight correction, based on the current set up where it requires a user who is admin, the sidekiq ui path will only work the tenant level:  http://library.localhost:8080/sidekiq.

However, it will work on both root and tenant level if we don't require  admin which will be to use only this:
 mount Sidekiq::Web => '/sidekiq'